.pylintrc

```diff
@@ -8,7 +8,7 @@ init-hook='import sys; sys.path.append("../oppia_tools/google_appengine_1.9.67/g
 [MASTER]
 
 # Checks for correct docstring style
-load-plugins=pylint.extensions.docstyle
+load-plugins=pylint.extensions.docstyle, scripts.explicit_kwargs_checker
```

Comment:
We should not have to modify this file every time we add a new checker. (In general, we should not need to modify _anything_ every time we add a new checker.)

Perhaps a way around this is to have all the checkers in one file?

Reply:
Yes, you're right! This sounds much better. Should I rename this script and the test script to `custom_lint_checks.py` and `test_custom_lint_checks.py`?

Reply:
Yes, those are good names.